app/cypress/e2e/cif/project-revision/index.cy.js

```diff
@@ -227,7 +227,36 @@ describe("the new project page", () => {
     // additional funding sources
     cy.findByText(/Additional Funding Source 1/i).should("be.visible");
 
-    //TODO: TEIMP Agreement, when fixture is added
+    // Emissions Intensity Report
+    cy.findByRole("heading", { name: /Emissions Intensity Report/i }).click();
+    cy.findByRole("link", { name: /Emissions Intensity Report/i }).click();
+    cy.contains(/Report Due Date/i)
+      .next()
+      .contains(/Jun(\.)? 10, 2020/);
+    cy.contains("Received Date")
+      .next()
+      .contains(/Jun(\.)? 10, 2020/);
+    cy.contains("General Comments")
+      .next()
+      .should("have.text", "TEIMP report comments 1");
+
+    cy.checkEmissionIntensityReportSummaryForm(
```

Comment:
Why do we need assertions in addition to the check function?

Reply:
I did not include checks for the fields specific to reporting requirements in the "checkEmissionIntensityReportSummaryForm" function. The reason behind this is that I intended to use this function again on the project summary view. As a result, I had to include additional checks in this section.